magento/magento2
# Migrating Store Grid to UI Components

### Description

Fix sorting by columns in store listing.

I override function `joinGroupAndStore` in `\Magento\Store\Model\ResourceModel\Website\Grid\Collection`.
I also use UI component to render store listing instead of block.

### Fixed Issues (if relevant)

1. magento/magento2#7283: Insignificant stores sorting issue

### Manual testing scenarios

1. Login to Admin.
2. Go to Stores -> All Stores.
3. Sort by any column.
4. Sorting should work.

### Contribution checklist
